newIDE/app/src/MainFrame/EditorContainers/HomePage/index.js

```diff
@@ -189,8 +187,13 @@ export const HomePage = React.memo<Props>(
                       ref={buildSectionRef}
                       project={project}
                       canOpen={canOpen}
-                      onOpen={onOpen}
-                      onCreateProject={onCreateProject}
+                      onChooseProject={onChooseProject}
+                      onCreateBlank={onCreateBlank}
+                      onShowAllExamples={() => onCreateProject(null)}
```

Comment:
This seems weird, maybe just a naming issue

Reply:
Maybe
```suggestion
                      onShowAllExamples={() => onCreateProject(/*exampleShortHeader=*/null)}
```

to hint to the reader that this is opening the create project but without a select example?